lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -2549,6 +2549,18 @@ class MailboxDashBoardController extends ReloadableController {
 
   List<Identity> get listIdentities => _identities ?? [];
 
+  bool validateNoEmailsInTrashAndSpamFolder() {
+    return selectedMailbox.value != null
+      && (selectedMailbox.value!.isTrash || selectedMailbox.value!.isSpam)
```

Comment:
Should we separate these cases? Why there is `validateNoEmailsInTrashAndSpamFolder` but no `validateNoEmailsInSpamFolder`

Reply:
It will duplicate test conditions, so we don't need separate it.